tests/std/tests/P2278R4_basic_const_iterator/test.cpp

```diff
@@ -298,7 +298,89 @@ constexpr void instantiation_test() {
     instantiator::call<test_iterator<contiguous_iterator_tag, CanDifference::yes>>();
 }
 
-int main() {
-    static_assert((instantiation_test(), true));
+template <class T>
+struct tracking_input_iterator {
+public:

```

Comment:
Style suggestion: If it's going to have `private` data members, it's not much of a `struct` - it should be a `class`.